experimental/generation/generator/packages/library/templates/standard/en-us/form.en-us.lg

```diff
@@ -16,8 +16,8 @@
 # propertyValue(property, val)
 - ${template(`${property}_Value`, val)}
 
-# entityValue(property, val)
-- ${template(`${property}Entity_Value`, val)}
+# entityValue(entity)
+- ${value(entity.value)}
```

Comment:
Shouldn't we be dispatching on a template name defined by the entity type?

Reply:
You mean ${value(entity.value)} should be replaced with ${template(${entity.type}_Value, entity.value)} ?